packages/expo-document-picker/ios/EXDocumentPicker/EXDocumentPickerModule.m

```diff
@@ -164,4 +162,34 @@ - (void)documentPickerWasCancelled:(UIDocumentPickerViewController *)controller
   _reject = nil;
 }
 
++ (unsigned long long int)getFileSize:(NSString *)path error:(NSError **)error
```

Comment:
`long long int` is the same as `long long`
And why not `NSNumber`?

```suggestion
+ (unsigned long long)getFileSize:(NSString *)path error:(NSError **)error
```

Reply:
> Why not `NSNumber`?

cause `fileAttributes.fileSize` is `unsigned long long` and I don't want to box it in  the `getFileSize` function.